src/location.c

```diff
@@ -545,11 +545,11 @@ handle_start_in_thread_func (GTask *task,
           info = g_desktop_app_info_new (id);
           name = g_app_info_get_display_name (G_APP_INFO (info));
 
-          title = g_strdup_printf (_("Give %s Access to Your Location?"), name);
+          title = g_strdup_printf (_("Allow %s to Access Your Location?"), name);
           if (g_desktop_app_info_has_key (info, "X-Geoclue-Reason"))
             subtitle = g_desktop_app_info_get_string (info, "X-Geoclue-Reason");
           else
-            subtitle = g_strdup_printf (_("%s wants to use your location."), name);
+            subtitle = g_strdup_printf (_("%s wants to know the location of this device."), name);
         }
 
       body = _("Location access can be changed at any time from the privacy settings.");
```

Comment:
According to the mockups this should be replaced with the content of the subtitle and the subtitle should not exists.